src/ParseServer.js

```diff
@@ -323,6 +329,26 @@ class ParseServer {
     return parseServer.start(options, callback);
   }
 
+  /**
+   * @static
+   * Create an async parse server
+   * @param {ParseServerOptions} options the parse server initialization options 
+   */
+  static createAsync(options) {
```

Comment:
not sure of "create". We already have `ParseServer.start()`

I can suggest to use `ParserServer.startAsync()`. I'll also suggest to use JS doc to explain the clear difference between `start` and `startAsync`.

I think we should start to deprecate `start()`. It could save a lot of time to some new developers if they try to perform some code like:

```js
const server = ParseServer.start()
const query = new Parse.Query('XXXX')
// Query could fail since parse-server could be not totally initialized,
// it will be random and depend of network/cpu.
// it could be hard to debug for new developers.
```

start: will return a parse-server app not fully initialized
startAsync: will resolve once parse-server is fully initialized

Reply:
I have created the function `startApp` which starts Parse Server and returns the express app